docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -2523,4 +2523,134 @@ Enabled JSON-RPC APIs.
             "net": "1.0"
         }
     }
-    ``` 
\ No newline at end of file
+    ```
+    
+    
+## EEA Methods
+
+### eea_sendRawTransaction
+
+Creates a private transaction, which has already been signed, generates the transaction hash and submits it to the transaction pool, and returns the transaction hash.
+
+The signed transaction passed as an input parameter is expected to include the privateFrom, privateFor, and restriction fields.
+
+To avoid exposing your private key, create signed transactions offline
+and send the signed transaction data using `eea_sendRawTransaction`.
+
+
+**Parameters**
+
+`data` -  Signed transaction serialized to hexadecimal format. For example:
+
+`params: ["0xf869018203e882520894f17f52151ebef6c7334fad080c5704d77216b732881bc16d674ec80000801ba02da1c48b670996dcb1f447ef9ef00b33033c48a4fe938f420bec3e56bfd24071a062e0aa78a81bf0290afbc3a9d8e9a068e6d74caa66c5e0fa8a46deaae96b0833"]`
+
+!!! note
+   [Creating and Sending Transactions](../Using-Pantheon/Transactions.md) includes examples of creating signed transactions using the [web3.js](https://github.com/ethereum/web3.js/) library.
```

Comment:
There's no example on this page of how to create a signed transaction using the `privateFrom`, `privateFor`, and `restriction` fields.  An example of how to do this is needed. Are `privateFrom`, `privateFor', and `restriction`  additional fields in the `txObject` as shown in the existing example (../Using-Pantheon/Transactions/#sending-ether)?